docs/appendix/glossary/location.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Organization
+id: organization
+full_link:
+short_description: An organization is a group of one or more locations that helps you organize your fleet.
+aka:
+---
+
+An organization is a group of one or more locations that helps you organize your fleet.
+
+An organization is the highest level grouping in the Viam platform, which generally represents a company, or other institution.
+You can also use organizations for departments or other entities that can have one or more {{< glossary_tooltip term_id="location" text="locations" >}}.
```

Comment:
```suggestion
An organization is the highest level grouping in the Viam platform, which generally represents a company, or other institution.
Every {{< glossary_tooltip term_id="location" text="location" >}} is grouped into an organization.
You can also have organizations for departments or other entities, or for personal use.
```
I don't love calling it a group of one or more locations since you can have an empty organization, but also maybe that will make sense to people so we should leave it. IDK.

Reply:
I mean while you can that doesn't really make any sense and one is automatically created - so I think that's ok 😂